edb/ir/ast.py

```diff
@@ -981,6 +983,12 @@ class Call(ImmutableExpr):
     # See comment in schema/functions.py for more discussion.
     prefer_subquery_args: bool = False
 
+    # Any extra information useful for compilation of special-case callables.
+    extras: typing.Optional[dict[str, typing.Any]] = None
+
+    # Inline body of the callable.
+    body: typing.Optional[Set] = None
```

Comment:
So, this expression replaces the function call in backend compilation, which is effectively "inlining function in the frontend compilation".

Why cannot we inline this fully, so there `qlast.FuncCall` is compiled to the body expression, so just `irast.Expr`? Without a wrapping `irast.Call`?

Reply:
From the call: because we still need to compile the arguments.